armi/reactor/grids.py

```diff
@@ -621,6 +621,10 @@ class Grid:
 
     * Unit step calculations use dot products and must not be polluted by the bound
       indices. Thus we reduce the size of the unitSteps tuple accordingly.
+
+    .. impl:: ARMI supports a number of structured mesh options.
+       :id: IMPL_0_3_0
+       :links: REQ_0_3
```

Comment:
I just feel like for the initial sample ones of these we should put them right on the method that does a very specific implementation of a requirement